<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTaskTableView</class>
 <widget class="QWidget" name="frmTaskTableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmStrateWIdget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMenu {
    background-color: rgb(100,100,100); 
	margin:0px;
	padding:0px;
}

QMenu::item {
	padding: 2px 25px 2px 20px;
    font-size:12px;
    color:rgb(188,188,188);
    height:20px;
}

QMenu::item:selected {
    background: rgba(188, 188, 188, 150);
}

QMenu::separator {
    height: 1px;
    background: rgba(188, 188, 188, 80);
    margin-left: 10px;
    margin-right: 5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="TaskTableView" name="taskTableView1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TaskTableView" name="taskTableView2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnBgColor">
       <property name="text">
        <string>表格背景颜色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGridColor">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>表格线条颜色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHeaderBgColor">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>表头背景颜色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHeaderTextColor">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>表头文字颜色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNormalColor">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>正常状态颜色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelectColor">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>选中状态颜色</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TaskTableView</class>
   <extends>QWidget</extends>
   <header>tasktableview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
